packages/block-editor/src/components/convert-to-group-buttons/toolbar.js

```diff
@@ -0,0 +1,64 @@
+/**
+ * WordPress dependencies
+ */
+import { useDispatch, useSelect } from '@wordpress/data';
+import { switchToBlockType } from '@wordpress/blocks';
+import { ToolbarButton, ToolbarGroup } from '@wordpress/components';
+import { group } from '@wordpress/icons';
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import { useConvertToGroupButtonProps } from '../convert-to-group-buttons';
+import { store as blockEditorStore } from '../../store';
+
+function BlockGroupToolbar( { label = __( 'Group' ) } ) {
```

Comment:
We might want to provide context for the label - `_x( 'Group', 'verb' )`. The `ConvertToGroupButton` also does this.

Reply:
Good catch! Done in https://github.com/WordPress/gutenberg/pull/39710/commits/9e5c2132df956aee0f78ee1ad8bf2e3b5ee5bca2